frontend/www/js/omegaup/components/login/Signup.vue

```diff
@@ -63,7 +63,7 @@
 
         <div class="row justify-content-md-center">
           <div class="col-md-8">
-            <input v-model="checked" type="checkbox" />
+            <input v-model="checked" type="checkbox" class="introjstandc" />
```

Comment:
¿En este nombre de clase podemos utilizar las palabras completas en lugar de abreviaciones?
```suggestion
            <input v-model="checked" type="checkbox" class="introjs-terms-and-conditions" />
```

de esta forma es más claro saber para que se utilizará la clase.

Otra cosa: ¿se puede hacer que este paso se aplique a todo el div? Para que el usuario sepa que es lo que tiene que leer